linux_os/guide/system/software/integrity/fips/fips_custom_stig_sub_policy/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Implement STIG Sub Crypto Policy'
```

Comment:
Do you plan on expanding the description and rationale? Or adding fixtest / checktext?

Reply:
Sentence is done now.